SU-SWS/saa_victoria_subtheme
# ADAPT-530: Fixed glitch for underline on red banner button.

# READY FOR REVIEW

# Summary
- Fixes the button on a vivid red banner - white underline on the button should not be there regularly, only on hover

# Review By (Date)
- 7/30 

# Urgency
- bug fix, let's fix soon

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Have a in page banner without an image (red background) and add a CTA link, regular link, button you put in through the text area and a button through the button field. Check that all have the correct underlines, particularly that the buttons do NOT show an underline until they are hovered over.
4. check that the small text in the banner is not cut off (padding-top 1px fix)

![image](https://user-images.githubusercontent.com/12898896/88960104-44baaf00-d260-11ea-9188-b05c890350d5.png)


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket related to ADAPT-530
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
